behaviours/meme_rater.py

```diff
@@ -139,28 +188,36 @@ async def rate_meme(
             curr_ratings = (0, 0)
             entry_exists = False
 
-        avg_rating = min(
-            max(
-                0, (sum(ratings) + curr_ratings[0]) // (len(ratings) + curr_ratings[1])
-            ),
+        ratings_sum = sum([result.rate for result in rating_results])
+        ratings_count = len(rating_results)
+
+        new_rating_sum = ratings_sum + curr_ratings[0]
+        new_rating_count = ratings_count + curr_ratings[1]
+
+        avg_rating: int = min(
+            max(0, new_rating_sum // new_rating_count),
             10,
         )
 
-        if len(explanations) > 1:
-            str_explanations = "\n".join(["* " + s for s in explanations])
+        if ratings_count > 1:
+            str_explanations = "\n".join(["* " + s.explanation for s in rating_results])
         else:
-            str_explanations = explanations[0]
+            str_explanations = rating_results[0].explanation
 
         if entry_exists:
             await message.remove_all_reactions()
 
+        final_emoji = "🐱"
```

Comment:
Why assign this at all? We don't use this assignment due to the following branches being exhaustive

Reply:
`MemeStat` class had an emoji property, so this `final_emoji` was supplied to the emoji property.

It is now removed as emoji is now determined at run time.